src/components/Page.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/books';
+
+function Page() {
+  const { booksReducer } = useSelector((state) => state);
+  const { books } = booksReducer;
+  const [booksDisplay, setBooksDisplay] = useState(null);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (books !== booksDisplay) {
+      setBooksDisplay(books);
+    }
+  });
+
+  if (booksDisplay) {
+    return (
+      <div className="books-container">
+        {booksDisplay.map((element) => (
+          <div className="bookcard" key={element.title}>
+            <div className="book-left">
+              <h4 className="book_genre">{element.category}</h4>
```

Comment:
- [ ] The keys should be unique so that components maintain their identity across updates. I have encountered an error in the console. Please check the below image and fix it.

![image](https://user-images.githubusercontent.com/53019916/131838970-acb4a89e-e5d9-4174-a8c1-de77aff02e5f.png)